src/benchmark/Akka.Cluster.Benchmarks/Sharding/ShardMessageRoutingBenchmarks.cs

```diff
@@ -43,6 +45,50 @@ public class ShardMessageRoutingBenchmarks
         private IActorRef _batchActor;
         private Task _batchComplete;
 
+#if (DEBUG)
```

Comment:
What's this for?

Reply:
I was using this to debug the functionality of the benchmark (be able to attach breakpoints etc.) as per https://benchmarkdotnet.org/articles/guides/troubleshooting.html - if another approach is usually taken, happy to remove or change as needed

Thanks

Reply:
Anything different about code in this block vs. the `#else` block other than `async Task` vs. `void`? Was trying to spot some but none jumped out at me. The other statement, with the debugging config added, makes sense to me.

Reply:
Correct I was running into an issue with async code in debug mode - the benchmark was executing before the task completion. I can refactor for a cleaner approach if it’s better (still would need two methods I believe due to async signature)

Reply:
Got it. Thank you!

Nope, that's ok - just adding a comment to explain why it's there would be sufficient. We can do that in a subsequent PR.

Thank you for your contribution!